.github/workflows/test-pull-request.yml

```diff
@@ -0,0 +1,41 @@
+name: Test Pull Request
+
+on:
+  pull_request:
+    branches:
+    - main
+
+jobs:
+  test:
+    name: Acceptance Test
+    runs-on: ubuntu-22.04
+    steps:
+    - name: Setup Go
+      uses: actions/setup-go@v3
+      with:
+        go-version: stable
+
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    # https://github.com/docker/setup-qemu-action
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v1
```

Comment:
Are we using QEMU for the arm64 support? I believe GHA has runners for arm64 now, should we try to use those instead, so we don't need QEMU. Idk how much work this would be, if it's a lot (i.e. big change), feel free to ignore this comment.

Reply:
Interesting, I didnt know. This code is from jammy-tiny-stack. 
On ubi stack we also produce images for ppc64le and s390x, in that case I'm not sure if GHA are able to support it. It needs further investigation. Yeap lets leave it for now, as it seems like an improvement.